modules/claims_api/app/controllers/claims_api/v2/veterans/disability_compensation_controller.rb

```diff
@@ -63,8 +73,18 @@ def get_pdf_data
           }
         end
 
-        def evss_service
-          ClaimsApi::EVSSService::Base.new(request)
+        def get_auth_token
+          bd_client = get_benefits_documents_service_client
+          call_benefits_documents_token_service(bd_client)
+        end
+
+        def get_benefits_documents_service_client
+          ClaimsApi::BenefitsDocumentsService::Service.new.get_client
+        end
+
+        # This will either get a new token or return the existing one for this service
+        def call_benefits_documents_token_service(bd_client)
+          ClaimsApi::BenefitsDocumentsService::Token.new(bd_client).get_token('bds_ccg')
         end
```

Comment:
confused on the duplication here. we should be able to just call the Token service and since it should inherit, we wouldn't need the first method?

Reply:
Yep I see it, updating it.